### PR TITLE
Translation of D-Dagen

### PR DESCRIPTION
**Pull request by alexjondiaz**

### CODE REVIEW

**Comment by bullfest** [comment]
This page should be checked against the current Swedish version to be totally sure it's up to date. It's really important that the business pages are correct

**Comment by alexjondiaz** [comment]
Updated to reflect the current page.

**Comment by LudwikJaniuk** [comment]
Looks much better, but the title for "Cerise package" should probably be "head sponsor" or the equivalent translation, right?

File: en/industry/d-dagen/body.md [hunk 1]
```diff
@@ -1,71 +1,64 @@
...
+*   Electricity and Internet connection
 *   Fika
-*   2 månaders annonsering på [djobb.se](http://djobb.se)
 
-**Pris:** 25 000 kr
+**Price:** 25 000 kr
 
 
-## Cerisa Paketet
+## The Cerise Package
```
Comment (bullfest) [inline en/industry/d-dagen/body.md]:
```suggestion
## Main Sponsor
```

File: en/industry/d-dagen/body.md [hunk 2]
```diff
@@ -1,71 +1,64 @@
...
+*   One table
+*   Four breakfast and lunch coupons
+*   Four dinner tickets
+*   Ten drink coupons
+*   Electricity and Internet connection
 *   Fika
-*   2 månaders annonsering på [djobb.se](http://djobb.se)
 
-För mer information och pris kontakta D-Dagenansvarig Linn Jensen på [d-dagen@d.kth.se](mailto:d-dagen@d.kth.se)
+For more information regarding the Cerise Package and costs, please contact Nils Streijffert at [d-dagen@d.kth.se](mailto:d-dagen@d.kth.se).
```
Comment (bullfest) [inline en/industry/d-dagen/body.md]:
```suggestion
For more information regarding the Main Sponsor Package and costs, please contact Nils Streijffert at [d-dagen@d.kth.se](mailto:d-dagen@d.kth.se). 
```

**Pull request merged**